create_ceph_cluster_on_centos.adoc

```diff
@@ -0,0 +1,122 @@
+// vim: tw=79
+= Provisioning Ceph Cluster On CentOs
+
+This document describes the mechanism to create ceph cluster on CentOS nodes
+
+
+== Install OS on Each Virtual Machines
+ 
+ * Provisioner Node : CentOs 7.0
+ * Mon              : CentOs 7.0
+ * Osd's            : CentOs 7.0
+
+== Configure Provisioner Node
+ * Installing ansible in provisioner node using epel repo. 
+   Run the following command to install the ansible:
+```
+wget http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-8.noarch.rpm
+rpm -ivh epel-release-7-8.noarch.rpm
+```
+ * Install ansible:
+```
+yum install ansible
+```
+ * Update the node:
+```
+yum update
+```
+
+== Configure Mon and Osd nodes :
+ * wget http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-8.noarch.rpm
+ * rpm -ivh epel-release-7-8.noarch.rpm
+ * rpm -Uvh https://download.ceph.com/rpm-jewel/el7/noarch/ceph-release-1-0.el7.noarch.rpm
+ * yum install ceph
+
+== Configuring Firewall
+ * As root, start the firewalld service, enable it to run on boot, and ensure that it is running:
```

Comment:
It would be nice to refer to ceph docs which describes firewall setup. Adding a link you used to write this up would be enough.